web/src/app/abstract-directives/list.directive.ts

```diff
@@ -56,7 +56,9 @@ export abstract class ListDirective extends BaseDirective implements OnInit, OnD
       const list = data.results;
       this.data = new MatTableDataSource<any>(list);
       this.changeCount(data.count);
-      this.listItemEvt.emit({ cmd: 'onLoad', row: list[0] });
+      if (list.length) {
```

Comment:
Is data.results allready array? Why result has any type? May be need `Array.isArray(list)` to condition?

Reply:
Yes. It would be more accurate